xarray/core/rolling_exp.py

```diff
@@ -8,8 +8,15 @@
 from xarray.core.computation import apply_ufunc
 from xarray.core.options import _get_keep_attrs
 from xarray.core.pdcompat import count_not_none
-from xarray.core.pycompat import is_duck_dask_array
-from xarray.core.types import T_DataWithCoords, T_DuckArray
+from xarray.core.types import T_DataWithCoords
+
+try:
+    import numbagg
+    from numbagg import move_exp_nanmean, move_exp_nansum
+
+    has_numbagg = numbagg.__version__
+except ImportError:
+    has_numbagg = False
```

Comment:
Is this a lazy import? Does our tests catch it?

Could you add numbagg here so we don't regress with regards to #6726:

https://github.com/pydata/xarray/blob/e8be4bbb961f58ba733852c998f2863f3ff644b1/xarray/tests/test_plugins.py#L216C9-L239

You can also use this for checking module availability:

https://github.com/pydata/xarray/blob/e8be4bbb961f58ba733852c998f2863f3ff644b1/xarray/namedarray/utils.py#L81-L84

Reply:
It's not a lazy import per se (it used to be, but I'm not sure that was necessarily deliberate vs. coincidental)

Using `module_available` doesn't seem to get the version?

What's the standard way of getting an optional module and checking the version in xarray? I'm happy to abide by the standard if there is one. The current approach does seem to work well in isolation...